admin-dev/themes/new-theme/js/pages/product/edit/manager/feature-values-manager.ts

```diff
@@ -108,48 +108,50 @@ export default class FeatureValuesManager {
       const $featureValueSelector = $(ProductMap.featureValues.featureValueSelect, $collectionRow).first();
       $featureValueSelector.prop('disabled', hasCustomValue);
       if (hasCustomValue) {
-        $featureValueSelector.val('');
+        $featureValueSelector.val('0');
```

Comment:
this way when its submitted it triggers "invalid option" error, or else I think it is considered as a placeholder value and doesn't trigger that error (though also not sure as its quite tricky :smile: )